docs/accessibilityinfo.md

```diff
@@ -119,52 +113,37 @@ class AccessibilityStatusExample extends Component {
   }
 
   componentWillUnmount() {
-    AccessibilityInfo.removeEventListener(
-      'reduceMotionChanged',
-      this._handleReduceMotionToggled
-    );
-
-    AccessibilityInfo.removeEventListener(
-      'screenReaderChanged',
-      this._handleScreenReaderToggled
-    );
+  this.reduceMotionChangedSubscription.remove();
```

Comment:
Is the formatting off on this?

Reply:
Whoops, yes. It is… thanks for catching that.